src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Conversion.cs

```diff
@@ -626,6 +643,20 @@ private BoundExpression MakeImplicitConversion(BoundExpression rewrittenOperand,
             bool @checked,
             bool explicitCastInCode,
             NamedTypeSymbol rewrittenType)
+        {
+            (LocalSymbol temp, BoundAssignmentOperator assignmentToTemp, ImmutableArray<BoundExpression> fieldAccesses) =
+                RewriteTupleConversionCore(syntax, rewrittenOperand, conversion, @checked, explicitCastInCode, rewrittenType);
+            var result = MakeTupleCreationExpression(syntax, rewrittenType, fieldAccesses);
+            return _factory.MakeSequence(temp, assignmentToTemp, result);
+        }
+
+        private (LocalSymbol temp, BoundAssignmentOperator assignmentToTemp, ImmutableArray<BoundExpression> fieldAccesses)
+        RewriteTupleConversionCore(SyntaxNode syntax,
```

Comment:
>RewriteTupleConversionCore [](start = 8, length = 26)

Feels like we could inline this method back into `RewriteTupleConversion`, because it only is used there and there is only one return statement.